client/src/ReferralLinkGenerator.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Card, Button } from 'semantic-ui-react'
+import { toast } from 'react-semantic-toasts'
+
+/**
+ * Button group that allows a user to copy a referral link to share
+ * @param {*} props
+ * schoolId - id for the user current school
+ * schoolName - name of the user school
+ * userId - userId for the student / alumnus 
+ */
+export default function ReferralLinkGenerator (props) {
```

Comment:
perhaps a future suggestion, but it would be nice to maybe have a modal where the user can enter the email of the person they are referring. That way we can keep activation energy lower by keeping it in app.